tests/components/template/test_light.py

```diff
@@ -172,12 +331,291 @@ async def async_setup_light(
     await hass.async_block_till_done()
 
 
+async def async_setup_legacy_format_with_attribute(
+    hass: HomeAssistant,
+    count: int,
+    attribute: str,
+    attribute_template: str,
+    optimistic_config: dict,
```

Comment:
Why is this parameter named `optimistic_config`, there's nothing locking this helper to configuring only lights with optimistic state, right? Same comment for the other helper functions.

Reply:
renamed to extra_config